### DISCUSSION TITLE
jSystemThemeDetector used by Document Archiver

### DISCUSSION THREAD

**Discussion by stephansann** [Dansoftowner/jSystemThemeDetector#5] [Show and tell]
Hi Daniel

First of all thank you very much for this wonderful library 🙏🏻
And also for PDFViewerFX.
I am using both in my project "Document Archiver" (https://github.com/Document-Archiver/com.sophisticatedapps.archiving.document-archiver).
Certainly I gave credit to you in the README 🙂

You offered to mention projects that are using this library under "Projects using jSystemThemeDetector".
So if you find the time, I would be happy if you would.

Thanks again and best regards
Stephan

**Comment by Dansoftowner** [comment]
Hi @stephansann ,

Wow, what a nice work!! 
Document Archiver is a really cool application, I feel honored you used `jSystemThemeDetector` and `PDFViewerFX` in it! 

I mentioned your project in the `README` both in [jSystemThemeDetector](https://github.com/Dansoftowner/jSystemThemeDetector#projects-using-jsystemthemedetector) and [PDFViewerFX](https://github.com/Dansoftowner/PDFViewerFX#projects-using-pdfviewerfx).

I referred to your github repository there, but if you want your website to be referred (https://document-archiver.github.io/) in the readmes please let me know!

Best regards
Daniel

**Reply by stephansann** [reply]
Hi again, Daniel
Thank you very much for the nice feedback and the inclusion in your READMEs 🙏🏻
A link to either place is fine - the project page has a prominent link to the website.
Best regards
Stephan
